TreeTracker/Models/Planter.swift

```diff
@@ -21,4 +21,17 @@ protocol Planter: class {
     var acceptedTerms: Bool { get set }
 }
 
-extension PlanterDetail: Planter { }
+extension PlanterDetail: Planter {
```

Comment:
I made this extension function that was available in `TreeService` available to all usages of `PlanterDetail`. This seemed like the more appropriate way to handle this.